src/bundle/Resources/views/themes/admin/ui/field_type/preview/content_fields.html.twig

```diff
@@ -433,10 +433,12 @@
                         <td>{{ 'ezimage.master_dimensions'|trans|desc('Master dimensions') }}:</td>
                         <td>{{ 'ezimage.width_and_height'|trans({'%width%': field.value.width, '%height%': field.value.height})|desc('Width: %width%px height: %height%px') }}</td>
                     </tr>
-                    <tr class="ez-field-preview__meta-value-row">
-                        <td>{{ 'ezimage.ratio'|trans|desc('Ratio') }}:</td>
-                        <td>{{ (field.value.width/field.value.height)|round(2) }}</td>
-                    </tr>
+                    {% if (field.value.height is not same as '0') %}
```

Comment:
Is `field.value.height` a string? Can we avoid string comparison here?

Reply:
I think that Twig actually casts variables to numbers when using `==`. 
I'm not 100% sure, but e.g `'0.0' == 0` is _true_ in Twig. So maybe we can just do this: 🤔 
```suggestion
                    {% if field.value.height == 0 %}
```

Reply:
And I wonder, maybe _Ratio_ field should still be visible but its value instead should be set to something like `n/a`? But I'm don't know if it would make sense, just asking :)

Reply:
Resolved in https://github.com/ezsystems/ezplatform-admin-ui/pull/2117/commits/8108b918551f27dcef98453cc8dcafce504ddc3f